nixos/modules/hardware/video/high-dpi.nix

```diff
@@ -0,0 +1,16 @@
+{ lib, pkgs, config, ...}:
+with lib;
+
+{
+  options.hardware.video.high-dpi.enable = mkEnableOption "Font/DPI configuration optimized for High-DPI displays";
+
+  config = mkIf config.hardware.video.high-dpi.enable {
+    console.font = lib.mkDefault "latarcyrheb-sun32";
```

Comment:
Could we use terminus here? It actually makes use of the additional pixels rather than taking a smaller font and naively scaling it up, which is nicer to use and also makes a better first impression.

Reply:
fixed.

Reply:
Terminus does not support unicode by the way, it did not support umlauts for example.

Reply:
Terminus has fairly wide character support; it just depends on the exact font you pick, per this table from the Terminus `README` (sadly not included in our package, it seems):

```
2.4. Legend.

names	mappings		covered codepage(s)

ter-1*	iso01, iso15, cp1252	ISO8859-1, ISO8859-15, Windows-1252
ter-2*	iso02, cp1250		ISO8859-2, Windows-1250
ter-7*	iso07, cp1253		ISO8859-7, Windows-1253
ter-9*	iso09, cp1254		ISO8859-9, Windows-1254
ter-c*	cp1251, iso05		Windows-1251, ISO8859-5
ter-d*	iso13, cp1257		ISO8859-13, Windows-1257
ter-g*	iso16			ISO8859-16
ter-h*	cp1255, iso08		Windows-1258, ISO8859-8
ter-i*	cp437			IBM-437
ter-k*	koi8r			KOI8-R
ter-m*	mik			Bulgarian-MIK
ter-p*	pt154			Paratype-PT154
ter-u*	koi8u			KOI8-U

ter-v*	all mappings / code pages listed above and many others, about 110
	language sets, 8 or 16 foreground colors depending on the kernel and
	console driver versions

names	weight

ter-*n	normal
ter-*b	bold
ter-*v	CRT VGA bold
```

Another native 16x32 option in nixpkgs with basic Unicode support is `spleen`.

Reply:
The font that was used before was more blurry than this Terminus. I have not tested `spleen` yet.

Reply:
For Terminus, `ter-132.psf.gz` would be a reasonable choice. The current is smaller than `latayrcyrheb-sun32` and focuses on Ukrainian language support.